interop.py

```diff
@@ -153,22 +153,32 @@ def _run_testcase(self, server: str, client: str, test: Callable[[], testcases.T
       "REQUESTS=\"" + reqs + "\" "
       "docker-compose up --abort-on-container-exit --timeout 1"
     )
-    output = subprocess.run(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
-    logging.debug("%s", output.stdout.decode('utf-8'))
+
+    status = TestResult.FAILED
+    output = ""
+    expired = False
+    try:
+      r = subprocess.run(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, timeout=60)
+      output = r.stdout
+    except subprocess.TimeoutExpired as ex:
+      output = ex.stdout
```

Comment:
This seems strange. Why do you need to re-assign output?

Reply:
It's not a reassignment. `subprocess.run` throws the exception, so the second line of the `try` block is never run.